ci/unit/test_main_node.sh

```diff
@@ -48,4 +48,4 @@ fi
 cd "$DAOS_BASE"
 sudo mount -t tmpfs -o size=16G tmpfs /mnt/daos
 IS_CI=true OLD_CI=false RUN_TEST_VALGRIND="$WITH_VALGRIND" \
-    DAOS_BASE="$DAOS_BASE" utils/run_test.sh
+    DAOS_BASE="$DAOS_BASE" ci/unit/run_test.sh
```

Comment:
did you move this file?

Reply:
Yes.  I started from the premise of doing "ls utils" and moving anything that was unlinky to be called directly by developers.  For run_test.sh I was not sure about it and planning to ask if people still call this or not, happy to back out this part of the change if you think that people still do.

Reply:
Put this back now to de-risk this.